app/containers/ITunesContainer/tests/selectors.test.js

```diff
@@ -1,18 +1,47 @@
-import makeSelectITunesContainer, { selectITunesContainerDomain } from '../selectors';
+import {
+  selectITunesContainerDomain,
+  selectITunesSearchQuery,
+  selectITunesData,
+  selectITunesError
+} from '../selectors';
+import { initialState } from '../reducer';
 
 describe('ITunesContainer selector tests', () => {
   let mockedState;
+  let searchQuery;
+  let iTunesData;
+  let iTunesError;
 
   beforeEach(() => {
+    searchQuery = 'Infinity';
```

Comment:
Why do we do this since we are not mutating the mockedState

Reply:
okay yes. I can just put it outside instead of `beforeEach()`.
that was not needed from my part. Thank you for pointing it out